tests/integration/telemetry/common/stats.go

```diff
@@ -138,6 +144,10 @@ func TestStatsFilter(t *testing.T, feature features.Feature) {
 							return err
 						}
 
+						if err := ValidateBucket(c, prom, "a", expectedBuckets); err != nil {
```

Comment:
nit: `"a"` can be clt.Config().Service()? Otherwise its hard to follow what `"a"` means

Reply:
updated, PTAL.